FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -552,6 +552,57 @@ await this.SetCustomUserClaimsAsync(uid, claims, default(CancellationToken))
             return userManager.ListUsers(options);
         }
 
+        /// <summary>
+        /// Generates the out-of-band email action link for password reset flows for the specified
+        /// email address.
+        /// </summary>
+        /// <exception cref="FirebaseAuthException">If an error occurs while setting custom claims.</exception>
+        /// <param name="email">The email of the user whose password is to be reset.</param>
+        /// <returns>A task that completes with the password reset link.</returns>
+        public async Task<string> GeneratePasswordResetLinkAsync(string email)
+        {
+            return await this.GeneratePasswordResetLinkAsync(email, null)
+                .ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Generates the out-of-band email action link for password reset flows for the specified
+        /// email address.
+        /// </summary>
+        /// <exception cref="FirebaseAuthException">If an error occurs while setting custom claims.</exception>
+        /// <param name="email">The email of the user whose password is to be reset.</param>
+        /// <param name="settings">The action code settings object which defines whether
```

Comment:
I think this technically should be "that" -- you're referring to a particular object that has these attributes, and not just adding information about the object.

Reply:
Done